### PR TITLE
#310 - Allow for topic/patterns with '#' and end of line comments

### PR DESCRIPTION
**Pull request by nMoncho**
Solves #310

### CODE REVIEW

File: broker/src/test/java/io/moquette/spi/impl/security/ACLFileParserTest.java [hunk 1]
```diff
@@ -43,10 +43,13 @@ public void testParseValidComment() throws ParseException {
         assertTrue(authorizations.isEmpty());
     }
 
-    @Test(expected = ParseException.class)
```
Comment (andsel) [inline broker/src/test/java/io/moquette/spi/impl/security/ACLFileParserTest.java]: Comment lines (like in Mosquitto conf) must start with # this test verifies that if there a space a before the file is invalid.

Comment (andsel) [reply broker/src/test/java/io/moquette/spi/impl/security/ACLFileParserTest.java]: This invalidate the PR

Comment (nMoncho) [reply broker/src/test/java/io/moquette/spi/impl/security/ACLFileParserTest.java]: I reverted to invalidate comments lines that start with a space. Thought that would be more fault tolerant if comment lines starting with spaces would be treated the same as without them.

**Comment by andsel** [comment]
Thanks @nMoncho merged
 Andrea

**Pull request merged**